### ISSUE TITLE
Feature Request: Add More Effects Custom Scripts Can Do

### ISSUE THREAD

**Issue by ebiggz** [Firebottle/Firebot#349]
It would be cool if Custom Scripts could initiate more Effects other than just chat messages. 

Thinking we could update the return object to contain an "effect" parameter that would look something like:

```javascript
{
    effect: {
        type: "show_image",
        metadata: {
            filepath: "C:\asdasd\asd\etc.png",
            location: "Middle",
            //etc
        }
    }
}
```

then the type and meta could vary depending on the effect.
The most used effects would probably be:
- chat
- show image
- play sound
- delay

We should probably allow the scripts to return a list of effects so one script could initiate multiple effects.

**Comment by ebiggz** [comment]
I've done some work on this. The custom script processor automatically detects if the script is using the original `run` method or if they are using the new "v2" version. It now accepts a list of effects in the response, currently the only one implemented is the "chat" effect.

**Comment by Firebottle** [comment]
Released in the dank update 4.2.0

**Comment by ebiggz** [comment]
Documentation for this: 
https://github.com/Firebottle/Firebot/wiki/Writing-Custom-Scripts
